### PR TITLE
XCM Configuration Composable LAYR Parachain #cu-20465559 #CEN-24

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
for https://app.clickup.com/t/20465559/CEN-24
I started to split fees per chain as Composable has these different. Also made as stub for assets registry in RFC013 until it merged.
So Composable has XCM enabled.  Next PRs will test this out.

### CODE REVIEW

File: code/parachain/runtime/composable/src/fees.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	rational,
+};
+
+use primitives::currency::CurrencyId;
+
+pub struct WellKnownForeignToNativePriceConverter;
+impl ForeignToNativePriceConverter for WellKnownForeignToNativePriceConverter {
+	fn get_ratio(asset_id: CurrencyId) -> Option<Rational64> {
+		match asset_id {
+			CurrencyId::xcDOT => Some(rational!(2143 / 1_000_000)),
```
Comment (blasrodri) [inline code/parachain/runtime/composable/src/fees.rs:13]: where does this number come from?

Comment (dzmitry-lahoda) [reply code/parachain/runtime/composable/src/fees.rs:13]: just something not to spam network. created task to find out and set release values https://app.clickup.com/t/8669gtt7j

File: code/parachain/runtime/common/src/fees.rs [hunk 2]
```diff
@@ -105,16 +98,24 @@ impl<AssetsRegistry: AssetRatioInspect<AssetId = CurrencyId>>
...
 #[cfg(test)]
 mod commons_sense {
-	use crate::fees::WellKnownForeignToNativePriceConverter;
+	use composable_traits::rational;
+	struct Dummy {}
+	impl ForeignToNativePriceConverter for Dummy {
+		fn get_ratio(asset_id: CurrencyId) -> Option<Rational64> {
+			match asset_id {
+				CurrencyId::USDT | CurrencyId::USDC => Some(rational!(15 / 1_000_000_000)),
+				CurrencyId::KSM => Some(rational!(375 / 1_000_000)),
```
Comment (blasrodri) [inline code/parachain/runtime/common/src/fees.rs:114]: Do we add LAYR and DOT here too?

Comment (dzmitry-lahoda) [reply code/parachain/runtime/common/src/fees.rs:114]: that is just for test. that ratios are sane.

**Review by obsessed-cake — APPROVED** [review]

**Comment by dzmitry-lahoda** [comment]
@Mergifyio refresh

**Comment by dzmitry-lahoda** [comment]
Works 
```
[2023-02-14T12:16:21Z TRACE tracing::span::active] <- get_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span] -- get_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span] twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span::active] -> twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span::active] <- twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span] -- twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span] twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span::active] -> twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span::active] <- twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span] -- twox_128_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span] set_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span::active] -> set_version_1;
[2023-02-14T12:16:21Z TRACE state] method="Put" ext_id=b961 key=45323df7cc47150b3930e2666b0aa31362f8058e9dc65b738fce4a22e26fa4f2 value=Some(05000000) value_encoded=0105000000
[2023-02-14T12:16:21Z TRACE tracing::span::active] <- set_version_1;
[2023-02-14T12:16:21Z TRACE tracing::span] -- set_version_1;
ok

test result: ok. 41 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 32.75s

   Doc-tests local-integration-tests

running 1 test
test src/testing.rs - testing::assert_lt_by (line 30) ... ignored

test result: ok. 0 passed; 0 failed; 1 ignored; 0 measured; 0 filtered out; finished in 0.00s

dz@dz-pc-11:~/github.com/ComposableFi/composable$ 
```

**Pull request merged**